uat/features/stanford_sites_helper.feature

```diff
@@ -1,9 +1,10 @@
+@stanford
 Feature: Stanford Sites Helper Module
   In order to ensure that upgrades do not break existing functionality
   As an administrative user
   I want to ensure the Stanford Sites Helper module is working properly
 
-  @live
+  @safe @deploy
```

Comment:
Ah, nice. This captures the difference between `@live`, `@safe`, and `@deploy` nicely.